models/prices/prices_native_tokens.sql

```diff
@@ -78,6 +78,7 @@ FROM
     ('fuse-fuse-network', null, 'FUSE', null, null),
     ('zel-zelcash', null, 'FLUX', null, null),
     ('myria-myria', null, 'MYRIA', null, null),
-    ('mode-mode', null, 'MODE', null,null)    
+    ('mode-mode', null, 'MODE', null,null),
+    ('sei-sei', null, 'SEI', null,null)
```

Comment:
By the way, `SEI` has been defined at L50 already: https://github.com/duneanalytics/spellbook/blob/333f6c98d371e60b9a87ae00458e5571c49b8ed7/models/prices/prices_native_tokens.sql#L50